prboom2/src/CMakeLists.txt

```diff
@@ -312,6 +312,8 @@ target_include_directories(prboom-plus PRIVATE
 target_link_libraries(prboom-plus PRIVATE
     ${OPENGL_LIBRARIES}
     ${SDL2_LIBRARIES}
+    winmm
+    comctl32
```

Comment:
This is not the right place for Windows specific libraries. As you can see, it broke Linux and macOS targets.

Reply:
https://github.com/coelckers/prboom-plus/pull/37/commits/780c682cc2de4056a71816f4958cfe78dec47ff5#diff-ec649d8d689ac310801f57cae7a3f446R316